<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TRunnerAnalysisChartsDialog</class>
 <widget class="QDialog" name="TRunnerAnalysisChartsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Runner Chart</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="12,1">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="chartVLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOptions">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#gbOptions {
     border: 1px solid gray;
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="lyRunnerInfo" stretch="1,1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lblChooseRunner">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Selected Runner </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbChartRunnerSelect">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblUpdateInterval">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Chart Update Interval</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbChartDIsplayInterval">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="lyXAxisOptions" stretch="1,1,0,0">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X-Axis Options</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbXAxisDisplay">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y-Axis Options</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbYAxisDisplay">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="lyYAxisOptions" stretch="0,0,0">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkHeikinAshi">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Heikin-Ashi Candles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
